styleguide/progress.scss

```diff
@@ -0,0 +1,46 @@
+@import 'colors';
+@import 'typography';
+
+// progress bars
+.kiln-progress-wrapper {
+  bottom: 100%;
```

Comment:
bottom: 100% pushes it above the enclosing element, rather than just sitting at the top :-)